src/plugins/console/public/styles/_app.scss

```diff
@@ -151,29 +151,44 @@
   max-width: 100%;
 }
 
-.monaco__status_badge--primary {
-  @extend %monaco__status_badge;
-  background-color: $euiColorVis1;
+.monaco__status_code_line--primary {
+  background-color: transparentize($euiColorVis1, .8);
 }
 
-.monaco__status_badge--success {
-  @extend %monaco__status_badge;
-  background-color: $euiColorSuccess;
+.monaco__status_code_line_number--primary {
+  background-color: transparentize($euiColorVis1, .4);
 }
 
-.monaco__status_badge--default {
-  @extend %monaco__status_badge;
-  background-color: $euiColorLightShade;
+.monaco__status_code_line--success {
+  background-color: transparentize($euiColorSuccess, .8);;
 }
 
-.monaco__status_badge--warning {
-  @extend %monaco__status_badge;
-  background-color: $euiColorWarning;
+.monaco__status_code_line_number--success {
+  background-color: transparentize($euiColorSuccess, .3);;
 }
 
-.monaco__status_badge--danger {
-  @extend %monaco__status_badge;
-  background-color: $euiColorDanger;
+.monaco__status_code_line--default {
+  background-color: transparentize($euiColorLightShade, .8);
+}
+
+.monaco__status_code_line_number--default {
+  background-color: transparentize($euiColorLightShade, .4);
+}
+
+.monaco__status_code_line--warning {
+  background-color: transparentize($euiColorWarning, .8);
+}
+
+.monaco__status_code_line_number--warning {
+  background-color: transparentize($euiColorWarning, .4);
+}
+
+.monaco__status_code_line--danger {
```

Comment:
When we have an error response the http error code seems to blend with the background and is not really visible unless selected with the cursor

<img width="827" alt="Screenshot 2024-09-17 at 08 34 39" src="https://github.com/user-attachments/assets/2bc41517-77d9-4588-8294-ac44c5c9e1c2">
<img width="818" alt="Screenshot 2024-09-17 at 08 34 43" src="https://github.com/user-attachments/assets/cb89605b-2452-42d4-86d2-a8157477b08d">

Reply:
Thanks for the review! This should be because the square parentheses are missing in the second request around [500 error] so the line wasn't tokenized correctly. Fixed in [Improve color contrast and fix a test](https://github.com/elastic/kibana/pull/192888/commits/fd60c2a2532d487861b3cf9cfacc07a8174d5536).